plugins/kubernetes-backend/schema.d.ts

```diff
@@ -61,5 +61,14 @@ export interface Config {
       apiVersion: string;
       plural: string;
     }>;
+    apiVersionOverrides?: {
+      pods?: string;
+      services?: string;
+      configmaps?: string;
+      deployments?: string;
+      replicasets?: string;
+      horizontalpodautoscalers?: string;
+      ingresses?: string;
```

Comment:
perhaps these should be limited to specific strings?

Reply:
Could also have made this either just

```ts
apiVersionOverrides?: { [pluralKind: string]: string };
```

or extensibly


```ts
apiVersionOverrides?: {
    pods?: string;
    services?: string;
    configmaps?: string;
    deployments?: string;
    replicasets?: string;
    horizontalpodautoscalers?: string;
    ingresses?: string;
  }
  & { [pluralKind: string]: string };
```

Just as an idea. It sounds cumbersome to maintain these explicit kind lists as we extend things in the future.